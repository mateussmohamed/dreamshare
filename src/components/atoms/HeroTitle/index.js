import React from 'react'
import styled from 'styled-components'

const HeroTitleStyled = styled.h1`
  ${props => props.theme.typography.montserratStyle};
  font-size: 52px;
  font-weight: bold;
  text-align: center;
  color: #ffffff;
  margin: 0;

  @media only screen and (min-width: 320px) {
    font-size: 30px;
  }

  @media only screen and (min-width: 768px) {
    font-size: 52px;
  }
`

const HeroTitle = props => (
  <HeroTitleStyled {...props}>{props.children}</HeroTitleStyled>
)

export default HeroTitle
