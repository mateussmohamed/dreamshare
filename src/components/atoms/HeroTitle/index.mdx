---
name: HeroTitle
menu: Atoms
---

import { Playground, PropsTable } from 'docz'
import HeroTitle from './index'

# HeroTitle
<PropsTable of={HeroTitle} />

<Playground style={{ backgroundColor: 'tomato', padding: 40 }}>
  <HeroTitle>
    SHARE YOUR HOLIDAY DREAM
  </HeroTitle>
</Playground>
