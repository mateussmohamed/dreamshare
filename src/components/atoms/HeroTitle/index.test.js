import React from 'react'
import { shallow } from 'enzyme'

import HeroTitle from './index'

test('[<HeroTitle />]', () => {
  const wrapper = shallow(<HeroTitle />)

  expect(wrapper).toMatchSnapshot()
})
