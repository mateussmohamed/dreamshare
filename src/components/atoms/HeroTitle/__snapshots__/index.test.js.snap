// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`[<HeroTitle />] matches to snapshot 1`] = `
<HeroTitle__HeroTitleStyled>
  I am a Title
</HeroTitle__HeroTitleStyled>
`;
